ui/src/overview.tsx

```diff
@@ -569,6 +574,24 @@ function ParticipantsGroupSection(props: {
               boxShadow: (theme) => `inset 0 1px 0 ${theme.palette.divider}`,
             }}
           >
+            <Button
```

Comment:
Make this an `IconButton` and put it beside the section name like the cohort rename button.